cli/cmd/build.go

```diff
@@ -28,7 +28,6 @@ var buildCmd = &cobra.Command{
 			checklist.DockerInstalled,
 			checklist.TerraformInstalled,
 			checklist.AwsInstalled,
-			checklist.AwsSessionManagerPluginInstalled,
```

Comment:
Another improvement would be not to require session manager when running on EKS.

Reply:
true